client/landing/gutenboarding/onboarding-block/acquire-intent/vertical-select/style.scss

```diff
@@ -106,6 +107,7 @@
 			width: 100%;
 			overflow: hidden;
 			text-overflow: ellipsis;
+			line-height: 1em;
```

Comment:
@andrewserong This should probably be replicated for site title placeholder or use a common classname for both https://github.com/Automattic/wp-calypso/pull/41971#discussion_r422066672

Reply:
Good idea! I've done some light refactoring to use a common classname, thanks for the reminder!

In the process, I also fixed up the site title placeholder so that the bottom border is aligned with the border when you start entering text in that field, too.